src/api/app/models/local_build_result/for_package.rb

```diff
@@ -0,0 +1,41 @@
+class LocalBuildResult
+  class ForPackage
+    include ActiveModel::Model
+    attr_accessor :package, :project, :show_excluded
+    attr_reader :excluded_results_count
+
+    def all
```

Comment:
Hmm :thinking: What I don't like now is that you always have to call ``all`` first so that you get the correct result for ``excluded_results_count``.

What about having e.g. a callback (http://api.rubyonrails.org/classes/ActiveModel/Callbacks.html) which does everything what is in ``all`` and you store the result in an instance variable instead.

What I mean is for instance:
Move the content of all to a ``reload`` function which gets called by an after initialize callback. The result in of reload you store in the instance variables ``excluded_results_count`` and ``all`` (don't like this name for a variable, maybe you come up with a better name)...

Sorry, sometimes you have to see everything altogether :cry: